ClassCompass/CanvasAPIClient.swift

```diff
@@ -120,16 +123,35 @@ class CanvasAPIClient {
         performRequest(url: self.canvasAPIURL + "/courses/\(courseId)/assignments", headers: self.defaultHeaders) { result in
             switch result {
             case .success(let assignmentRaw):
-                var assignments : [Assignment] = []
-                for assignmentRaw in assignmentRaw {
-                    let assignment = Assignment(id: assignmentRaw["id"] as! Int,
-                                                name: assignmentRaw["name"] as! String,
-                                                dueDate: self.stringtoDate(assignmentRaw["due_at"] as! String),
-                                                description: assignmentRaw["description"] as! String,
-                                                grade: 0)
-                    
-                    self.database.saveAssignment(assignment)  // Save each assignment to the database
-                    assignments.append(assignment)
+                var assignments: [Assignment] = []
+                for assignmentDict in assignmentRaw {
+                    guard let id = assignmentDict["id"] as? Int else {
+                        print("Error: 'id' is not an Int or is missing in assignment: \(assignmentDict)")
+                        continue
+                    }
+                    guard let name = assignmentDict["name"] as? String else {
+                        print("Error: 'name' is not a String or is missing in assignment: \(assignmentDict)")
+                        continue
+                    }
+                    guard let description = assignmentDict["description"] as? String else {
```

Comment:
I get an error on some descriptions. For now, I would just assign "" and comment the actual value. I need to figure out why